### PR TITLE
Hotfix/age check

### PR DESCRIPTION
**Pull request by Kelvin120**

### CODE REVIEW

**Review by robinfredriks — APPROVED** [review]

File: Festivity/MenuFunction.cs [hunk 1]
```diff
@@ -401,7 +401,7 @@ public static void Menu(string[] consoleOptions, object[] objects = null)
                     case "Order Tickets":
                         Console.Clear();
                         CatalogPage.activeScreen = false;
-                        if (LoggedInAccount.User.AccountID == 0)
+                        if (LoggedInAccount.IsLoggedIn() == false)
```
Comment (EnnyGit) [inline Festivity/MenuFunction.cs]: IsLoggedIn() return a boolean, no need to compare it to a boolean 😁

File: Festivity/TicketBuy.cs [hunk 2]
```diff
@@ -27,32 +30,79 @@ private static void UpdateCurrentFestivalTickets(int festivalId)
             }
         }
 
-        public static void TicketShow()
+        public static bool AgeCheck(int festivalId) //Checks if the user is old enough to use the program
```
Comment (EnnyGit) [inline Festivity/TicketBuy.cs]: Old enough to use the program? 😢

File: Festivity/TicketBuy.cs [hunk 3]
```diff
@@ -27,32 +30,79 @@ private static void UpdateCurrentFestivalTickets(int festivalId)
...
+                    if (AgeCheck(currentFestivalId))
+                    {
+                        UpdateCurrentFestivalTickets(currentFestivalId);
+
+                        MenuFunction.option = 0;
+
+                        while (true)
+                        {
+                            List<string> menuOptionsList = new List<string>();
+                            string line = "----------------------------------------------------------------------";
```
Comment (EnnyGit) [inline Festivity/TicketBuy.cs:78]: Duplicate code is bad 😜

**Review by EnnyGit — CHANGES_REQUESTED** [review]

File: Festivity/TicketBuy.cs (hunk 2 above)
Comment (Kelvin120) [reply Festivity/TicketBuy.cs:33]: Het is maar een comment bruh

File: Festivity/TicketBuy.cs (hunk 3 above)
Comment (Kelvin120) [reply Festivity/TicketBuy.cs:78]: ??

File: Festivity/MenuFunction.cs (hunk 1 above)
Comment (Kelvin120) [reply Festivity/MenuFunction.cs]: Fixed

File: Festivity/TicketBuy.cs (hunk 3 above)
Comment (Kelvin120) [reply Festivity/TicketBuy.cs:78]: I legit don't know which code you're referring to.

If you're talking about "if (currentFestivalId == festival.FestivalID)" (line 69)
This is needed for "festival.FestivalAgeRestriction" (line 100)

File: Festivity/TicketBuy.cs (hunk 2 above)
Comment (Kelvin120) [reply Festivity/TicketBuy.cs]: Fixed...

File: Festivity/TicketBuy.cs (hunk 3 above)
Comment (Kelvin120) [reply Festivity/TicketBuy.cs:78]: Fixed

**Pull request closed without merging**